movie/src/components/HumbergerNavbar/HumbergerNavbar.css

```diff
@@ -0,0 +1,75 @@
+.navbarBuger {
+  position: absolute;
+  background-color: rgb(176, 223, 244);
+  z-index: 5;
+  border-radius: 5px;
+}
+
+.navbarBugerShow {
+  display: block;
+}
+
+.navbarBugerHide {
+  display: none;
+}
+
+.dropdownBuger {
+  float: left;
```

Comment:
please dont use `float`. Use `flex` instead

Reply:
Yes, I have changed it to flex instead.